src/SprykerMiddleware/Zed/Process/Communication/Console/ProcessConsole.php

```diff
@@ -29,13 +29,19 @@ class ProcessConsole extends Console
     protected const OPTION_ITERATOR_LIMIT = 'limit';
     protected const OPTION_LOG_LEVEL = 'flagLogLevel';
     protected const OPTION_INPUT = 'input';
+    protected const OPTION_INPUT_OPTIONS = 'input-options';
     protected const OPTION_OUTPUT = 'output';
+    protected const OPTION_OUTPUT_OPTIONS = 'output-options';
     protected const OPTION_PROCESS_NAME_SHORTCUT = 'p';
     protected const OPTION_ITERATOR_OFFSET_SHORTCUT = 's';
     protected const OPTION_ITERATOR_LIMIT_SHORTCUT = 'l';
     protected const OPTION_LOG_LEVEL_SHORTCUT = 'f';
     protected const OPTION_INPUT_SHORTCUT = 'i';
     protected const OPTION_OUTPUT_SHORTCUT = 'o';
+    protected const OPTION_INPUT_OPTIONS_SHORTCUT = 't';
+    protected const OPTION_OUTPUT_OPTIONS_SHORTCUT = 'u';
+
+    protected const DEFAULT_OPTIONS = '{}';
```

Comment:
should we always depend on Json format?

Reply:
We should pass here named options. Console API can receive either string or array, but if it is array then it is numered array not associative. So IMO JSON format here is the most convenient solution. If you have any concerns here I'll be happy to know it.

Reply:
Here is how `--help` output look like:
![image](https://user-images.githubusercontent.com/44202033/60179266-fc8fb880-9825-11e9-93fe-9e90c3b19651.png)